### ISSUE TITLE
bitfinex2 - missing markets

### ISSUE THREAD

**Issue by s5s** [ccxt/ccxt#1051]
Hi, the command ccxt.bitfinex2().load_markets().keys() does not return all the current markets trading on bitfinex. For example, "YYW/USD" is a market available on bitfinex but not through ccxt.

At least the following coins are missing: YYW, SNT, QASH, GNT

- OS: Linux Mint 18.2 Sonya
- Programming Language: Python 3.6
- CCXT version: 1.10.556 (updated though python's pip a few minutes ago)
- Exchange: bitfinex2
- Method: load_markets()

```

python << EOF
import ccxt
print(sorted(ccxt.bitfinex2().load_markets().keys()))
EOF

```

**Comment by kroitor** [comment]
@s5s hi! Thank you for reporting, this will be fixed very soon.

**Comment by s5s** [comment]
This is not isolated to bitfinex, I don't think. I find similar issue on binance (is that the case?). Would the fix target the root cause for all exchanges or is it exchange particular?

**Comment by kroitor** [comment]
@s5s should be ok with Binance. So far this is a known case of Bitfinex2, because it has hardcoded symbols, but we are going to rework it shortly.

**Comment by MichaelYusko** [comment]
@kroitor Someone working on this ticket? if no, I can look into.

**Comment by kroitor** [comment]
@MichaelYusko Thanks! The task is to implement fetchMarkets() for Bitfinex2, I was going to do that, but if you can submit a pull-request on this you will save me some time for other tasks. That would be awesome! )

Main considerations:
- we have to follow these rules exactly/literally: 
    - https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
    - https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes
- the easiest way to approach this task is to check the implementation of Bitfinex v1 fetchMarkets() + the same method for other major exchanges and combine them into an implementation for v2

Let me know if you are going to give it a try. Thx!

**Comment by MichaelYusko** [comment]
@kroitor Got it, but one little moment, I'll able to look into, and make a PR, since the next Thursday/Friday, if it works for you, I'll do it.

**Comment by kroitor** [comment]
@MichaelYusko no worries then, I hope to fix it in next few hours when I get to it, and will update you here, of course.

**Comment by MichaelYusko** [comment]
@kroitor Cool, so I'll look into issues, since the next Friday;) and will help you to solve an issue.

Happy holidays!

**Comment by kroitor** [comment]
@MichaelYusko happy holidays to you! Thx!

**Comment by jjhesk** [comment]
@kroitor this symbol api is missing.. https://api.bitfinex.com/v2/symbols_details is that the correct api? 1.10.804

**Comment by kroitor** [comment]
@jjhesk that link doesn't seem to work for me...

**Comment by jjhesk** [comment]
i think the api is blocked again.

**Comment by kroitor** [comment]
Fixed in #1459. Thanks, @jjhesk
